webpack/components/EditableTextInput/EditableTextInput.js

```diff
@@ -27,6 +38,18 @@ const EditableTextInput = ({
   const onEditClick = () => {
     setEditing(true);
     if (setCurrentAttribute) setCurrentAttribute(attribute);
+
+    if (isPassword) {
+      if (passwordPlaceholder) {
+        setPasswordPlaceholder(null);
+        setPasswordReset(true);
+      }
+
+      if (inputValue === null) {
```

Comment:
I wonder if this piece should be moved to the onEdit handler function in CdnConfiguration? ie is this behavior specific to my use case here for CdnConfiguration, or should it be generalized for all password usage of this component?

Reply:
I'd say probably yes, it should be moved. The only way it will be null is if the parent component supplies `value={null}` via the prop, so that's pretty specific.